packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -265,7 +265,6 @@ export interface ISegment extends IMergeNodeCommon, Partial<IRemovalInfo>, Parti
 	 * This is defined if and only if the insertion of the segment is pending ack, i.e. `seq` is UnassignedSequenceNumber.
 	 * Once the segment is acked, this field is cleared.
 	 *
```

Comment:
nit: rather than remove, might be better to put these in a `\@privateremarks` block? That should make them still visible to merge-tree devs without leaking the info to API surface / docs.